src/index.js

```diff
@@ -0,0 +1,48 @@
+import './css/style.css';
+import Weather from './javascript/weather';
+
+const weather = new Weather();
+
+if (navigator.geolocation) {
+  navigator.geolocation.getCurrentPosition(weather.currentLocation);
+}
+
+function permissionHanldler() {
```

Comment:
### Uses `ES6` syntax 

* [ ] Kindly ensure that you are using ES6 syntax consistently, all across the codebase as explained in this ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)). Here you are supposed to use arrow function instead of the `function` keyword in this  `permissionHanldler` declaration.

```suggestion
const permissionHanldler = () => {
```